source/Camera/Maintenance/VerifyCameraTrueness/VerifyCameraTrueness.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+This example shows how to verify the local dimension trueness of a camera.
+If the trueness is much worse than expected, the camera may have been damaged by
+shock in shipping in handling. If so, look at the CorrectCameraTrueness sample.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iomanip>
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        // For convenience, print the timestamp of the latest correction
```

Comment:
https://github.com/zivid/zivid-cpp-samples/pull/148/files#r533510067

Reply:
I'll fix the tense. I think it's appropriate with a comment here, though, since it's kind of on the side of the main purpose of the sample.